include/rocksdb/env.h

```diff
@@ -948,6 +962,28 @@ class FileLock {
   void operator=(const FileLock&);
 };
 
+class DynamicLibrary {
+public:
+  typedef void *(*FunctionPtr)();
```

Comment:
Just curious. Due to name demangling, loading a C++ function is not trivial. I noticed that you used `extern C` on those functions. Do you think we should make this typedef more specific about this type of functions ,e.g. 'CLinkageFunctionPtr`?

Reply:
nvm, this typedef is class-scoped, leaving it as it is now is fine with me.